README.md

```diff
@@ -230,20 +154,10 @@ Then open your browser and view [file:///tmp/fullsite/index.html]
 
 Contributing
 ------------
-1. Fork it
-2. Clone the repo (`git clone --recursive <REPO_URL>/CONNECT.git`)
-3. If you already have the repo and need to update to the latest, use `git pull`
-4. Create a branch (`git checkout -b my_feature`)
-5. Commit your changes (`git commit -am "Added new feature"`)
-6. Push to the branch (`git push origin my_feature`)
-7. Open a [Pull Request][]
-
-[Pull Request]: https://github.com/CONNECT-Solution/CONNECT/pulls
-[Download Maven]: http://maventest.apache.org/download.html
-[Install Maven]: http://maventest.apache.org/download.html#Installation
-[Eclipse]: http://www.eclipse.org/downloads/
-[ant 1.7.1]: http://archive.apache.org/dist/ant/binaries/apache-ant-1.7.1-bin.zip
-[egit plugin]: http://www.eclipse.org/egit/download/
-[m2eclipse plugin]: http://eclipse.org/m2e/
-[MySQL 5.1.x]: http://dev.mysql.com/downloads/mysql/5.1.html
 
+Please checkout [code contribution](https://connectopensource.atlassian.net/wiki/display/CONNECTWIKI/How+to+Contribute+Code) for guidelines about how to contribute.
+
+License
+-------
+
```

Comment:
This license link is wrong, use: https://connectopensource.atlassian.net/wiki/display/CONNECTWIKI/CONNECT+License+Agreement

Also move the license up to below the CONNECT description.